Updated automation-update-account-powershell.md

AakashMasand opened the pull request:
Changed the value of $KeyCredential in automation-update-account-powershell.md as the script was failing because PSADKeyCredential has been now moved to Microsoft.Azure.Graph.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AakashMasand committed:
Updated automation-update-account-powershell.md

Changed the value of $KeyCredential in automation-update-account-powershell.md as the script was failing because PSADKeyCredential has been now moved to Microsoft.Azure.Graph.
